modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -24,82 +24,80 @@ This page describes the options on the *Deploy Application* panel that is identi
-* You cannot move an application into the same environment it's in already. Runtime Manager prevents you from duplicating applications in a single environment. If you wish to have the same, or similar applications in a single environment, modify one application's file to distinguish it from the original.
-
-
-
-
-
-
+* To avoid domain name conflict, an application with the same name cannot be deployed in two environments at the same time. Modify the application's name slightly to deploy it to another environment.
+* Runtime Manager prevents you from duplicating applications in a single environment, so you cannot move an application into the environment it's already in. To have the same, or similar, applications in a single environment, modify one application's file to distinguish it from the original.
 
 
 === From Anypoint Studio
 
-You can easily deploy your applications to CloudHub, straight from Anypoint Studio. This is specially helpful if you're still developing the application and want to deploy it often to an online test environment.
+You can deploy your applications to CloudHub from Anypoint Studio. This is helpful if you're still developing the application and want to deploy it often to an online test environment.
+
+. In Anypoint Studio open your application as a Mule Project.
 
-. With your application open in Anpoint Studio as a Mule Project, Right-click on the project node in the package explorer. Then select *Deploy to Anypoint Platform* > *Cloud* from the cascading menu.
+. In *Package Explorer*, right-click the project node, then select *Anypoint Platform* > *Deploy to CloudHub*.
 +
-image::deploy-to-cloudhub.png[deploy+to+cloudhub]
+// image::deploy-to-cloudhub.png[deploy+to+cloudhub]
 +
-. If this is your first time deploying in this way, a popup menu asks you to provide your login credentials for the Anypoint Platform. Studio stores your credentials and uses them automatically the next time you deploy to CloudHub.
+. If this is your first time deploying from Studio, a popup dialog box asks you to provide your login credentials for Anypoint Platform. Studio stores your credentials and uses them automatically the next time you deploy to CloudHub.
 
 +
 [TIP]
 You can manage these credentials through the Studio *Preferences* menu, in *Anypoint Studio* > *Authentication*.
 
-. After you sign in, the Deploy Application menu opens.
+. After you sign in, the *Deploy Application* dialog window opens.
 +
 image::deployappfirstscreen.png[DeployAppFirstScreen]
 
+[NOTE]
+--
+You cannot deploy an application from Studio to the CloudHub *Design* environment. The *Design* environment is specific to applications created in Anypoint Platform Design Center.
+--
+
 [TIP]
 For an example of deploying to CloudHub directly from Anypoint Studio, see xref:general:getting-started:deploy-to-cloudhub.adoc[Deploy to CloudHub].
```

Comment:
The two tips and one note so close together makes me wonder if we need them all three to be called out? I'd make the 2nd tip just straight text. Also, the link in line 94 isn't rendering as a link in github. I can't see what's wrong...maybe links outside the repo don't render and it's okay?

Reply:
Might be a Github thing. The link does seem to work for me.